app/controllers/notes_controller.rb

```diff
@@ -0,0 +1,58 @@
+class NotesController < ApplicationController
+  before_action :set_note, only: [:show, :edit, :update, :destroy]
+
+  # GET /notes
+  def index
+    @notes = Note.all
+  end
+
+  # GET /notes/1
+  def show
+  end
+
+  # GET /notes/new
+  def new
+    @note = Note.new
+  end
+
+  # GET /notes/1/edit
+  def edit
+  end
+
+  # POST /notes
+  def create
+    @note = Note.new(note_params)
+
+    if @note.save
+      redirect_to @note, notice: 'Note was successfully created.'
```

Comment:
should we put these (as parameterized) strings into an I18n file? They will get used a lot.... And it might be a good intro to I18n concepts.